packages/ocl/src/Adaptor.js

```diff
@@ -110,82 +109,119 @@ function getMappingInfo(params) {
     });
 }
 
-// /**
-//  * Creates a fictional resource in a fictional destination system using a POST request
-//  * @public
-//  * @example
-//  * create("/endpoint", {"foo": "bar"})
-//  * @function
-//  * @param {string} path - Path to resource
-//  * @param {object} params - data to create the new resource
-//  * @param {function} callback - (Optional) callback function
-//  * @returns {Operation}
-//  */
-// export function create (path, params, callback) {
-//   return state => {
-//     path = expandReferences(path)(state)
-//     params = expandReferences(params)(state)
-
-//     const { baseUrl, username, password } = state.configuration
-
-//     const url = `${baseUrl}/${path}`
-//     const auth = { username, password }
-
-//     const config = {
-//       url,
-//       body: params
-//     }
-
-//     return http
-//       .post(config)(state)
-//       .then(response => {
-//         const nextState = {
-//           ...composeNextState(state, response.data),
-//           response
-//         }
-//         if (callback) return callback(nextState)
-//         return nextState
-//       })
-//   }
-// }
-
-// /**
-//  * Create a fictional patient in a fictional universe with a fictional REST api
-//  * @public
-//  * @example
-//  * createPatient({"foo": "bar"})
-//  * @function
-//  * @param {object} params - data to create the new resource
-//  * @param {function} callback - (Optional) callback function
-//  * @returns {Operation}
-//  */
-// export function createPatient (params, callback) {
-//   return state => {
-//     params = expandReferences(params)(state)
-
-//     const { baseUrl, username, password } = state.configuration
-
-//     const url = `${baseUrl}/patient`
-//     const auth = { username, password }
-
-//     const config = {
-//       url,
-//       body: params,
-//       auth
-//     }
-
-//     return http
-//       .post(config)(state)
-//       .then(response => {
-//         const nextState = {
-//           ...composeNextState(state, response.data),
-//           response
-//         }
-//         if (callback) return callback(nextState)
-//         return nextState
-//       })
-//   }
-// }
+/**
+ * Get a source repository in OCL
+ * @public
+ * @example
+ * getMappings(
+ *   "MSFOCG",
+ *   "lime-demo",
+ *   { page: 1, exact_match: "off", verbose: false },
+ *   (state) => {
+ *     // Add state oclMappings
+ *     const oclMappings = state.data;
+ *     return { ...state, data: {}, references: [], response: {}, oclMappings };
+ *   }
+ * );
+ * @function
+ * @param {string} owner - An OCL user or organization
```

Comment:
I think this is `ownerId`? Or `ownerName`? Just helps differentiate from some kind of owner object